### ISSUE TITLE
MediaBrowser.Server.Mono.exe not starting on OpenBSD

### ISSUE THREAD

**Issue by 0x6373** [MediaBrowser/Emby#2531]
I'm trying to run emby on OpenBSD (snapshot).
On each start it fails with the following error:
```
        System.AggregateException: One or more errors occurred. ---> System.ArgumentException: Value does not fall within the expected range.
          at System.Net.Sockets.Socket.SetSocketOption (System.Net.Sockets.SocketOptionLevel optionLevel, System.Net.Sockets.SocketOptionName optionName, System.Int32 optionValue) [0x00061] in <973e3e46ba0e49e99f857dcad1eb9286>:0
```
I'm using mono 4.6.2.6
Anybody got a hint on what could cause this?

[unhandled_b18f3faf-201e-4fe3-88db-abf762800f7a.txt](https://github.com/MediaBrowser/Emby/files/851343/unhandled_b18f3faf-201e-4fe3-88db-abf762800f7a.txt)
[server-63625371234.txt](https://github.com/MediaBrowser/Emby/files/851347/server-63625371234.txt)

**Comment by LukePulverenti** [comment]
It appears to be due to the system not supporting DualMode sockets for Ipv6, however, the mono version of BSD is throwing an ArgumentException when it's supposed to be throwing a SocketException like the other platforms do. We will have to adjust for the next release, thanks.

**Comment by 0x6373** [comment]
Thanks for the fast feedback.
Do you need someone to test before the release?

**Comment by 0x6373** [comment]
Has been fixed in release 3.2.8.4-beta

Working great.
